recipes/emi/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "emi" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{version}}
+
+source:
+  url: https://github.com/BioPhys-Turin/EfficientMutualInformation/archive/refs/tags/{{ version }}.tar.gz 
+  sha256: 1e64bef7f56a3c02a96b0e89ebc91706cf305471ce352766e9c7eb89ce5a3aa6
+
+build:
+  number: 0
+  skip: True  # [win or osx]
+
+requirements:
+  build:
+    - cmake
+    - make
+    - {{ compiler("cxx") }}
+    - {{ stdlib("c") }}
+  run:
+
+test: 
+  commands:
+    - hk --help
```

Comment:
Looks like this is not in the path!

```
+ hk --help
/home/conda/staged-recipes/build_artifacts/emi_1743773378542/test_tmp/run_test.sh: line 7: hk: command not found
```

Reply:
That should be correct :-/

Reply:
this failed https://github.com/conda-forge/staged-recipes/runs/39992846319

Reply:
and their are the same code

Reply:
also win is failing, but I have skip [win]

Reply:
Are you sure pipelines are ok?

Reply:
> May you help me, which is the correct path expected here?
> 
> https://github.com/BioPhys-Turin/EfficientMutualInformation/blob/a98abd3fdeb06e8363d911baea99888937ae34a3/build.sh#L18
> 
> I am copying the executable in `cp local/src/hk ${PREFIX}/bin/`

Oh! That PREFIX only exist here in staged-recipes. You should have at copy command as part of conda-forge's pipeline, not as part of the build upstream.

Reply:
now it passed the test

Reply:
without any change

Reply:
why is win failing? It must be skipped, I suppose

Reply:
Linux is still failing. You need to copy that binary to PREFIX here, where that env variable is defined.

Reply:
ok, https://github.com/conda-forge/staged-recipes/pull/29399/commits/620f4c22df5b04fcddfa69bc11756a053ae357ff implements this